Frontend/CO.CDP.OrganisationApp.Tests/Pages/Registration/OrganisationDetailsSummaryModelTest.cs

```diff
@@ -62,6 +63,105 @@ public async Task OnPost_OnSuccess_RedirectsToOrganisationOverview()
             .Which.PageName.Should().Be("/OrganisationOverview");
     }
 
+    [Fact]
+    public async Task OnPost_DuplicateOrganisationName_AddsModelError()
+    {
+        var problemDetails = OrganisationEntityFactory.GivenProblemDetails(statusCode: 400, code: ErrorCodes.ORGANISATION_ALREADY_EXISTS);
+        var aex = OrganisationEntityFactory.GivenApiException(statusCode: 400, problemDetails: problemDetails);
```

Comment:
If you imported the namespaces at the top with:

```csharp
using static CO.CDP.Organisation.WebApiClient.OrganisationEntityFactory;
```

this call could be reduced to a more expressive:


```csharp
 var aex = GivenApiException(statusCode: 400, problemDetails: problemDetails);
```